crates/experimental-api/src/gen.rs

```diff
@@ -1305,7 +1309,7 @@ pub mod gen {
         #[serde(default)]
         pub order: Option<i64>,
         #[serde(flatten)]
-        pub event: Event,
```

Comment:
this was another problem caused by the shadowing `EVENT` with `EVENT_CONTENT`...